### ISSUE TITLE
On Android, Clicking "Show data directory" crashes ZeroNet

### ISSUE THREAD

**Issue by styromaniac** [HelloZeroNet/ZeroNet#1744]
In case it's specific to ZeroNet-Kivy and doesn't belong here, yes, that's how I'm running ZeroNet.

**Comment by anoadragon453** [comment]
I don't believe the Android version has even received an update since the vulnerability list went out. @mkg20001 are you able to update the internal ZeroNet version without too much haslle?

**Comment by mkg20001** [comment]
@anoadragon453 Maybe. Building this app is very complicated, the build process really is comparable to Schrödinger's cat: It might not work tomorrow because of some tiny change somewhere that requires a fix in some other module.

**Comment by DaniellMesquita** [comment]
@mkg20001 
Could Cordova run Python code, to bring zites as dedicated apps?

**Comment by HelloZeroNet** [comment]
"Cordova: Mobile apps with HTML, CSS & JS"

Probably not as zeronet requires python

**Comment by DaniellMesquita** [comment]
> "Cordova: Mobile apps with HTML, CSS & JS"
> 
> Probably not as zeronet requires python

Electron also is HTML, CSS & JS but supports PY

**Comment by mkg20001** [comment]
@DaniellMesquita If you manage to get python itself running on android that should work. That's what python-for-android project does. But that's working without cordava, too.

**Comment by mkg20001** [comment]
@DaniellMesquita That's a completely different topic than what this issue is about. It's about a specific crash and you're asking if this thing runs in cordova/electron and if somebody could create a boilerplate for that. Create a different issue for that, please.
